portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -134,9 +179,11 @@ definition {
 			var iterator = MathUtil.sum("${iterator}", "1");
 		}
 
-		Workflow.assignBlogEntriesToUsers(
-			numberOfBlogInstances = "4",
-			numberOfUsers = "3");
+		// Workflow.assignBlogEntriesToUsers(
+		// 	numberOfBlogInstances = "4",
+		// 	numberOfUsers = "3");
+		WorkflowMetrics.executeWorkflowScript(scriptName = "assignOpenTasksToUsers.groovy");
```

Comment:
Thank you for fixing the macro!!

I think I have seen what I did wrong! I will fix it as soon as possible. Hopefully the way you fixed it will work on CI. Let's wait for ci:test:workflow to run.